backend/pharmacy/api/serializers.py

```diff
@@ -13,6 +23,12 @@ class MedicalSerializer(serializers.ModelSerializer):
     class Meta:
         model = Medical
         fields = '__all__'
+
+
+class MedicineSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Medicine
+        fields = '__all__'
```

Comment:
Yaha Kya ho raha hai

Reply:
Isme buss woh serializer ko bata raha h ki model Medicine ke saare fields ko serialize krna h api request me

Reply:
let's say Medicine model has `name`, `price` field... I'm just telling that utilize all the field, i.e both name and price and use them when giving a response or validate if every field is sent or not.

Reply:
ohhhhhh got it